src/themes/basic/elements/icon.variables

```diff
@@ -1,22 +1,22 @@
-/*-------------------
+/* -------------------
    Icon Variables
---------------------*/
+-------------------- */
 
-@fontPath  : "../../themes/basic/assets/fonts";
+@fontPath : "../../themes/basic/assets/fonts";
 
 @fonts: {
-  @solid: {
-    font-family: @fontName;
-    src: url("@{fontPath}/@{fontName}.woff") format('woff');
-    font-style: normal;
-    font-weight: @normal;
-    font-variant: normal;
-    text-decoration: inherit;
-    text-transform: none;
-  };
+    @solid: {
+        font-family: @fontName;
+        src: url("@{fontPath}/@{fontName}.woff") format('woff');
+        font-style: normal;
+        font-weight: @normal;
+        font-variant: normal;
+        text-decoration: inherit;
+        text-transform: none;
+    };
 };
-@icon-deprecated-map: {};
+@icon-deprecated-map: {}; // stylelint-disable-line indentation, block-closing-brace-newline-after
 @icon-outline-map: {};
```

Comment:
because of some stylelint issue, see https://github.com/stylelint/stylelint/issues/6514

thankfully, only a few (vs. all huge code base) `stylelint-disable-line` comments were needed

maybe it is also related with semicolon we keep for #1832

Reply:
As the stylhint bug does not get fixed offiicially, but i dont really want to blow up our code with thousands of extra hints just for the sake of having a linter for space intendation: Could you try to follow the advise they stated and fork/fix the rule in a custom rule?

I was really not amused when i saw that hundreds of extra comments which are only meant for the linter. We should avoid that and better disable the rule then  (which they also told you to do in their next major release

Reply:
only 12 files were affected, but I have disabled the `indentation` rule for now and will see if I can fix the rule in another PR easily

now no single extra/stylelint comment is needed 👍